includes/class-wc-payments-tasks.php

```diff
@@ -30,6 +30,10 @@ public static function init() {
 	 * Adds a task to the WC 'Things to do next' task list the if disputes awaiting response.
 	 */
 	public static function add_task_disputes_need_response() {
+		if ( ! WC_Payments::get_account_service()->is_stripe_account_valid() ) {
```

Comment:
Using `is_stripe_account_valid()` instead of `is_stripe_connected()` because `is_stripe_connected()` can return true when onboarding process started but not finished.

I recently found the issue due to that. See p1691736710053399-slack-C01B8KNUYSW and https://github.com/Automattic/woocommerce-payments/pull/6978.